src/core/api/Providers/ProductsProvider.ts

```
@@ -0,0 +1,27 @@
+    static createProduct(payload: object, query = {}) {
+        return api.post("/products", payload, { params: query });
+    }
+
+    static updateProduct(id: number, payload: object, query = {}) {
+        return api.put(`/products/${id}`, payload, { params: query });
+    }
+
+    static deleteProduct(id: number) {
+        return api.delete(`/products/${id}`);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)